---
title: "Bad working title: population genetics of independent gametophytes using genotyping-by-sequencing"
author: Aaron M. Duffy, Don R. Farrar, Paul G. Wolf
---

ABSTRACT
========

INTRODUCTION
============
The "typical" fern lifecycle consists of alternating independent generations of diploid sporophytes and haploid gametophytes. The sporophyte is generally larger, longer-lived and has a more complex structure than the gametophyte [@Sheffield1994-jv] and their drastically different morphologies can result in different environmental or habitat requirements. However, in order for the entire lifecycle to occur, the same site must be suitable for the spore to germinate, the gametophyte to survive to produce gametes, the gametes to unite, and the sporophyte to survive to produce spores. All of these diverse processes must be supported in one location, though not necessarily at the same time [@Peck1990-th]. Some fern species are capable of asexual reproduction and subsequent dispersal in one or both generations, allowing individuals, clones, or colonies of either sporophytes or gametophytes to persist in places where the other cannot. This ability for gametophytes to survive and spread, even if the habitat in a particular time or place is not suitable for sporophytes may play an important role in long-distance fern dispersal and colonization [@Sato1982-uh; @Dassler2001-zv; @Ebihara2013-iq].

It is not noteworthy to observe sporophytes and not find gametophytes growing at the same site (since the gametophytes tend to be shorter-lived, more difficult to identify, and frequently overlooked), so examples demonstrating differences in habitat requirements between life stages are typically gametophyte colonies found growing independent of sporophytes. For example, sporophytes and gametophytes of _Callistopteris baueriana_ (Endl.) Copel. are found in wet, high altitude forests of the Hawaiian islands, whereas, only gametophytes are found at drier, lower locations [@Dassler1997-vo]. Several other species have gametophytes found over a larger geographic or environmental range than the sporophytes, including _Trichomanes speciosum_ Willd. (Hymenophyllaceae) in the British Isles, Europe and Canary Islands [@Ratcliffe1993-rx; @Rumsey1998-yx; @Rumsey1999-au; @Kingston2005-wl], _Hymenophyllum wrightii_ Bosch (Hymenophyllaceae) on the northwest coast of North America [@Persson1958-vl; @Iwatsuki1961-wa; @Taylor1967-yn; @Duffy2015-ol], and _Crepidomanes schmidtianum_ (Zenker ex Taschner) K. Iwats. (Hymenophyllaceae) in Japan and Korea [@Lee2014-df]. All of these ferns have gametophytes that are long-lived and capable of clonal growth and asexual reproduction.
NOTE-Use terminology from [@Pinson2017-lc] "separation of generations"

_Crepidomanes intricatum_ (Farrar) Ebihara & Weakley (Hymenophyllaceae), found as gametophytes throughout the Appalachian Mountains in eastern North America is a particularly interesting example of this phenomenon because no sporophytes have ever been found. The gametophytes are single-cell wide, branching filaments that produce asexual gemmae and grow in tangled clusters ranging from a few individuals to dense colonies covering several square meters (Figure 1).

_C. intricatum_ has rbcL sequence identical to Asian _C. schmidtianum_ sporophytes [@Ebihara2008-sx], suggesting a close relationship, (relatively recently formed species or not a separate species or possibly the result of hybridization?) and it has been hypothesized that _C. intricatum_ may not be a separate species, but rather another example of a species with gametophyte range extending beyond the range of the sporophytes. It is not clear whether North American gametophytes are the same species or possibly the result of hybridization involving _C. schmidtianum_, and this relationship does not explain the apparent lack of ability to produce sporophytes in North America.

Citations for temperature moderated habitats of _C. intricatum_: Farrar 1998, Chambers and Emery 2016.

_D. petersii_ has been found growing up to 50km from the nearest sporophyte population [@Farrar1992-pm; @Bray1996-hp]. Even though the gametophytes of _D. petersii_ and _C. intricatum_ are identical, can both form independent gametophyte populations and have overlapping geographic ranges, only recently have genetic tools been used to determine the species identity of unknown gametophytes [@Pinson2017-eu]. ([@Farrar1992-pm] -- they used isozymes on 30 samples from east of the Mississippi river that did not have _D. petersii_ or _V. boschiana_ sporophytes and they all had the _C. intricatum_ banding pattern)
[@Bray1996-hp] for apparent example of assuming independent gametophyte populations are _D. petersii_ based on their presence in a region where the sporophytes have been identified.

[@Ebihara2009-sg] for example of complex hybridization relationships in independent gametophytes of _Vandenboschia_ species. They hypothesize that gametes from gametophytes that cannot produce sporophytes hybridize with another species. They found two samples that appeared to have 2 different taxa growing intertwined... so it isn't too far-fetched to think that some of my samples could contain individuals of 2 species.

The three species in this study are related closely enough that they were once all included in the genus _Trichomanes_ ([@Ebihara2006-ce; @Ebihara2007-kq] and their gametophytes cannot be distinguished morphologically.

The 5 _C. intricatum_ populations from two sites tested in Ehibara et al 2008 had no variation in 1206bp of rbcL. They differed from Asian _C. schmidtianum_ at 0-3 basepairs. We cannot say anything about whether there is rbcL variation within _C. intricatum_ (small sample size). There appears to be variation within _C. schmidtianum_ but we don't have enough info to understand how that variation is distributed. Other examples of ferns with North American / Japanese disjuncts have higher levels of rbcL sequence divergence consistent with separation "long before the last glacial age" while the divergence between _C. intricatum_ and _C. schmidtianum_ is more consistent with a recent separation.

Ebihara et al 2013- In a survey in Japan, where fern distributions are well mapped. Most cordiform gametophytes had sporophytes growing at the same site or had been reported within 10km. But non-cordiform gametophytes were often found >10km from the nearest known sporophytes and gametophytes for 3 species were found that are not known to occur as sporophytes in Japan.

Previous attempts to explore population genetics of independent gametophytes used isozymes or cpDNA. Farrar found no isozyme variation in _C. intricatum_, and we failed to find sufficiently variable cpDNA markers. Is this the first attempt to look at population genetics of a fern with independent gametophytes using DNA rather than allozymes? No. Rumsey 1996 used cpDNA in European T. speciosum.

From Pinson et al. 2017:
  "Currently, the assumption is that the distributions of widespread gametophyte-only taxa are the result of ancient spore dispersal that occurred before these ferns lost their sporophyte counterparts."

NOTES-Rumsey et al 1999 suggest that since habitat is sparsely distributed, "primary colonization" events are likely rare and if the time between arrival at a specific site of gemmae or spores is long, the ones that got to the site first could exclude secondary colonists. That doesn't seem to be the case for _C. intricatum_ but this is based on assumptions that are untested in _C. intricatum_. Rumsey et al 1998 looked at genetic diversity in T. speciosum at the northernmost limits of its range in Scotland and found no allozyme variation within populations (one reason why we assumed T. intricatum collections would be clones--they found no variation within sites) and several multilocus genotypes in the area. Most genotypes were limited to a few locations while 2 were widespread. In Europe (Rumsey et al 1998b) they did find variation within some ~5% of populations, and 11% of sites (their site and population definitions differ from mine) and one collection appeared to be a combination (their data better support it being an individual with a mix of the two phenotypes rather than a mix of individuals with different phenotypes) of 2 multi-locus phenotypes. Similar to what I found in some cases, they found some very dissimilar clones growing close together (within meters). They did an analysis similar to IBD and found no correlation of geographic proximity and phenotype similarity. Their results for central Europe are relevant to my study because this is the portion of T. speciosum's range where sporophytes are absent. In that region they found "considerable overall genetic variation" with variation "partitioned primarily between, not within, sites" and no IBD. They determined that the patterns of variation were compatible with 1) multiple independent colonization events, 2) sexual recombination among just a few colonists, or 3) a combination... so pretty much any of the possible hypotheses.
......


Dispersal of gemmae, fragments of gametophyte filament or even entire gametophytes.

  * What we know about _C. intricatum_ and other filmies in north america. Be sure to include something about how gametophytes found growing near sporophytes are assumed to be the same species as the sporophyte, whereas gametophytes found growing without sporophytes nearby are assumed to be _C. intricatum_ and individual patches are assumed to represent single clones.
  * Hypotheses for how _C. intricatum_ arrived at its current distribution
  * What we know about _C. intricatum_ genetics.
  * GBS
  * Goals of project:
    * Using genotyping-by-sequencing on independent gametophytes collected from throughout the Eastern US to 1) identify gametophyte samples to species and 2) explore the population genetics of _C. intricatum_ gametophytes at different scales
    * Evaluate hypotheses about how these populations were established and spread


METHODS
=======

## Sampling

50 suspected _C. intricatum_, _D. petersii_, or _V. boschiana_ gametophyte samples, one _D. petersii_ sporophyte sample, and three _V. boschiana_ sporophyte samples collected between 1965 and 2007 were selected to represent much of the known geographic range of _C. intricatum_ (Figure 3). Samples included gametophytes from previously glaciated areas, from multiple sites within some geographic regions, and from multiple patches within some sites. In addition, multiple DNA samples were extracted from some collections to look for variation within presumed clones (Table 1, Table 2).

## DNA extraction

Because of the small size of individual gametophytes and their tangled filamentous growth, each sample represents many individuals that were growing in close contact with each other (presumably but not necessarily clones) in <1 square-cm of silica-dried gametophyte mat. Bryophytes, lichen, sand, and other contaminants were manually separated from the gametophyte tissue under 20X magnification. DNA was extracted from 2--80 mg of cleaned tissue using a CTAB protocol [@Doyle1987-dp] with the following modifications to the cell disruption process. Samples were ground in CTAB buffer in 1.5mL tubes using a pestle attached to an electric drill on its slowest speed setting. While grinding, the tube was bathed in ice and pressure on the drill was applied in 1 second pulses to avoid heating the sample. For samples with less than 10 mg of tissue, 10--50 mg of autoclaved sand was added to improve grinding. Each sample was ground in three rounds of approximately 100 pulses with 100 uL of CTAB buffer in the first round and an additional 200uL added in subsequent rounds (500 uL total). After extraction the DNA was quantified using a Qubit fluorometer (Invitrogen, Carlsbad, California, USA).

## Library preparation and Illumina sequencing

The GBS library was prepared according to an established double-digest restriction site-associated DNA sequencing (ddRADseq) protocol [@Gompert2012-ir; @Parchman2012-qm] using restriction enzymes EcoR1 and Mse1. PCR amplification steps were performed using iProof high-fidelity DNA polymerase (New England Biolabs) to minimize PCR error. To maximize the number of homogeneous loci across samples, pooled samples were size selected using Blue Pippen (Sage Science, Beverly, Massachusetts, USA), retaining loci between 250--350 bp. Samples were run on a single lane of Illumina HiSeq 2500 with 100 bp single-end sequencing at the Genome Sequencing and Analysis Facility at University of Texas.

## Identifying loci and calling genotypes

Raw Illumina reads were inspected for quality using fastQC [@Andrews2010-yx] and analyzed with ipyRAD v.0.7.1 [@Eaton2014-rr] using default settings except as noted below. Reads were sorted by barcode allowing up to two mismatched bases in the barcode sequence, filtered to remove low quality bases and reads with excessive low quality bases, and trimmed to remove Illumina adapter sequences.

To determine the appropriate clustering threshold for _de novo_ assembly of loci, clustering was performed over a range of threshold values from 0.95 down to 0.80. Higher values risk splitting alleles of a single locus into separate clusters, whereas lower values risk combining paralogs into single clusters. At higher threshold values the average number of clusters in _V. boschiana_ sporophyte samples is higher than in suspected _V. boschiana_ gametophyte samples, as would be expected if the ploidy level of the sporophytes is greater than the gametophytes. As the threshold values decrease the average number of clusters in sporophyte and gametophyte samples become similar, suggesting that multiple alleles in the sporophyte samples are being combined into loci. A threshold value of 0.85 was selected as a high value where the number of clusters in _V. boschiana_ sporophyte and gametophyte samples are similar (Figure 4).

Because allozyme studies suggest that _C. intricatum_ gametophytes are diploid [@Farrar1985-nm], samples were treated as diploid for estimating heterozygosity and error rates, and genotype calling in ipyRAD.

Each 35--100 bp locus identified by ipyRAD may contain multiple SNPs. To avoid using tightly linked SNPs, ipyRAD provides output files with genotypes from 1 randomly selected SNP per locus (.ustr format). However, this random sampling takes place after loci have been filtered for minimum sample coverage, so some SNPs may not have the requested minimum sample coverage even though the locus they are within does, resulting in higher than expected levels of missing data. Also, because the SNP from each locus is selected randomly at the time the output file is generated, multiple ipyRAD datasets from runs with increasing minimum sample coverage values may result in datasets that are not subsets of each other. To ensure that all the analyses in this study were based on subsets of the same data and only included SNPs with the desired minimum sample coverage, we used ipyRAD to generate a dataset of loci with coverage in six samples and filtered that "base ipyRAD dataset" to the desired minimum sample coverages using a custom python script.

## Separating samples by species

To group samples by species, we generated three datasets from the base ipyRAD dataset including all 54 samples and loci present in at least 33%, 50%, and 66% of samples, to balance the risks of using large numbers of loci with much missing data against the risks of using very few loci with high sample coverage (Table 3). Each dataset was analyzed using Structure [@Pritchard2000-ek] with 3 clusters to determine whether each gametophyte sample:
  1. was consistently assigned to a cluster with the _V. boschiana_ sporophytes or _D. petersii_ sporophyte (suggesting that it is likely a gametophyte of one of those species),
  2. was consistently assigned to a cluster without sporophytes (suggesting that it is likely a gametophyte of _C. intricatum_), or
  3. was not consistently assigned to any one cluster (suggesting that it may include a mixture of more than one species of gametophyte, that it may contain hybrid gametophytes, or that there is not sufficient genetic signal to consistently make an assignment).

To compare with species assignments from Structure, neighbor-joining Euclidean distance trees were generated with the adagenet package in R [@Jombart2008-sq; @Jombart2011-zo] with bootstrap values calculated using the poppr R package [@Kamvar2014-de].

Mean observed heterozygosity levels in each species were compared using ANOVA with Tukey HSD post hoc comparisons, and loci were tested for deviation from Hardy Weinberg equilibrium in each species in R.

## Investigating genetic variation in _C. intricatum_

To explore genetic variation in _C. intricatum_, gametophyte samples assigned as _C. intricatum_ and gametophytes assigned as _C. intricatum_ or Unknown were further analyzed separately. Nine samples with low read counts (fewer than 1 million cleaned and quality filtered reads) were removed from the analysis to reduce the level of missing data and risk of mis-called genotypes due to missing alleles.

To determine if the genetic signal from _V. boschiana_ and _D. petersii_ in the potentially mixed Unknown samples could be removed bioinformatically, allowing the _C. intricatum_ genetic signal to be analyzed, loci were identified as "_Crepidomanes_ loci" or "Non-_Crepidomanes_ loci" based on whether they were present in any sample assigned as _D. petersii_ or _V. boschiana_. We hypothesized that removing these non-_Crepidomanes_ loci from a sample with mixed genetic signal would remove loci specific to _D. petersii_ or _V. boschiana_ as well as loci shared between those species and _C. intricatum_, leaving mostly loci unique to _C. intricatum_. Four _C. intricatum_ subsets of the data were generated from the base dataset, with and without Unknown samples, and with and without non-_Crepidomanes_ loci, each using a minimum sample coverage cutoff of 50% (Table 3). Patterns of genetic variation were analyzed using Structure, neighbor-joining Euclidean distance trees, and minimum spanning networks. Isolation by distance was tested using Mantel tests and Pearson's product-moment correlation with adagenet in R, and genetic distances were compared at different geographic scales and between glaciated and non-glaciated sites.

RESULTS
=======

## Illumina sequencing data

The sequenced dataset consisted of approximately 278 million reads. An analysis of the raw reads using fastQC [@Andrews2010-yx] raised no significant quality issues. Quality score ranges per base and per sequence were high (REPORT NUMBER) and the N content (ambiguously called bases) was low (REPORT NUMBER) over the length of the reads. Mean GC content was 42% and was consistent over the non-barcode portion of the reads. 85% of reads matched one of the expected barcodes (allowing up to 2 mismatches) with 4.4 million mean reads per barcode. Reads per sample ranged from 120000--11000000.

## Separating samples by species

Structure runs using all samples and loci with 33, 50, and 66% sample coverage resulted in similar sample groupings despite the differences in numbers of loci and levels of missing data in each dataset. Gametophyte samples that were consistently assigned to the group without sporophytes (probability of assignment >= 0.900 and lower bound of the confidence interval >= 0.800) in all three Structure analyses were classified as _C. intricatum_ (29 samples) for subsequent analyses. Gametophyte samples consistently assigned to a group containing sporophytes (probability of assignment >= 0.100) in all three datasets were assigned to the species of the sporophytes with which they grouped (6 _V. boschiana_, 3 _D. petersii_). All remaining gametophyte samples were assigned as "Unknown" (12 samples)(Table 1, Figure 5). Several of the Unknown samples have low numbers of reads or loci, which may be causing them to be inconsistently assigned in the three structure runs. However two samples with large numbers of reads consistently had large proportions of their ancestry assigned to both _C. intricatum_ and _V. boschiana_, and two had large proportions assigned to both _C intricatum_ and _D. petersii_, suggesting that at least some of the Unknown samples might contain a mixture of gametophyte species or gametophytes produced by hybrid sporophytes.

Neighbor-joining Euclidean distance trees generally support the gametophyte species assignments based on Structure, with samples for each species grouped together (Figure 6). There is also some support for the subgroupings of Unknown samples. One clade includes all of the _D. petersii_ samples and the two Unknown samples suspected to be mixed _C. intricatum_/_D. petersii_, and another clade contains all of the _V. boschiana_ samples, both of the samples suspected to be _C. intricatum_/ _V. boschiana_, and all but one of the remaining Unknown samples.

Analyses of observed heterozygosity levels and deviation from Hardy Weinberg equilibrium support differences between samples classified as different species as well. Approximately 28% of loci in _C. intricatum_ samples and 4% of loci in Unknown samples were not in Hardy Weinberg equilibrium, whereas none of the loci in _D. petersii_ or _V. boschiana_ samples were identified as deviating from HWE. Mean observed heterozygosity was higher in _C. intricatum_ samples (29%) than _D. petersii_ (12%), _V. boschiana_ (8%), or Unknown samples 12%), and this difference was statistically significant (F=33.91, df=3, p<2e-16).

## Investigating genetic variation in _C. intricatum_

### _C. intricatum_ and Unknown samples

After removing samples with less than 1 million reads, 25 _C. intricatum_ and 7 Unknown samples remained, representing 14 sites in 5 states (Figure 3).

Removing the non-_Crepidomanes_ loci did not have a large effect on the total number of loci in the filtered dataset relative to the unfiltered dataset (5% of total loci removed; Table 3), likely because even a non-_Crepidomanes_ locus present in all of the Unknown samples would not meet the 50% minimum sample coverage cutoff. Individual samples differed in the proportion of loci with signal (not missing data) that were lost (Table 4), but Unknown samples lost an average of 20% of their loci with signal, whereas _C. intricatum_ samples only lost an average of 4%, and this difference is statistically significant (t=3.49, df=6.13, p-value = 0.012).

Analysis of Structure K-means and Evanno's delta-K suggest that the samples cluster optimally into 2 groups (Figure 7; Figure 8). Structure runs using the unfiltered subset at K=2 suggest that most of the Unknown samples cluster separately from the _C. intricatum_ samples (Figure 9a). Most of the _C. intricatum_ samples have the majority of their ancestry assigned to one cluster, whereas five of the seven Unknown samples have the majority of their ancestry assigned to the other cluster.  After filtering non-_Crepidomanes_ loci (Figure 9b) this apparent difference between _C. intricatum_ and Unknown samples remains. Only one of the suspected mixed samples (S07) noticeably changes ancestry assignments in a way consistent with a mixed sample having the non-_Crepidomanes_ signal removed. The other three suspected mixed samples (S13, S20, and S21) have over 99% of their ancestry assigned to the second cluster even after non-_Crepidomanes_ signal is removed. It is possible that S07 is actually a mixed sample and the others are not, but it is also possible that the attempt to bioinformatically remove non-_Crepidomanes_ genetic signal was not successful and that removing the loci present in the _V. boschiana_ and _D. petersii_ samples did not remove a large enough proportion of the non-_Crepidomanes_ signal. In either case, because we could not determine whether the Unknown samples cluster separately because of variation in _C. intricatum_ genetic signal or because of genetic signal from other species, we removed the 7 Unknown samples the remaining analyses focused only on the 25 _C. intricatum_ samples.

### _C. intricatum_ samples

The 25 _C. intricatum_ samples with over 1 million reads represent 12 sites in three states (Figure 3).

Removing the non-_Crepidomanes_ loci had even less effect on the _C. intricatum_ datasets than on the _C. intricatum_ + Unknown datasets, with only 2% of total loci removed (Table 3; Table 4). The results of analyses using unfiltered and filtered datasets were similar to each other and yield identical inferences, so only results for the unfiltered dataset are presented.

For the _C. intricatum_ samples the optimal number of clusters appears to be just one. There is no peak in the Evanno delta-K values (Figure 8) and estimated log likelihood values increase only slightly at low K-values before dropping at higher values (Figure 7). There is no clear geographic pattern to the ancestry assignments at K=2 (or higher values), with the ancestry for each sample divided between two clusters (Figure 10).   

Mantel tests identified evidence of isolation by distance (r~m~=0.271, p=0.02  with 999 permutations). Using Pearson's product-moment correlation, geographic and genetic distance are correlated and this correlation is significantly different from zero (r=0.271, t=3.986, df=284, p=8.553e-05).  However, the pattern of isolation by distance is not strong (Figure 11) and there is high variation in pairwise genetic distances at all geographic scales (Figure 12).

The Neighbor-joining Euclidean distance tree (Figure 13) and the Minimum spanning network (Figure 14) support the lack of genetic structure identified by Structure and weak isolation by distance. Tip branch lengths are long relative to internal branches in the distance tree and most samples do not group geographically. Samples from the same region, site, or even the same collection do not consistently form monophyletic groups or distinct clusters. The three Western Pennsylvania samples do group together, but distances between the samples (all from one site) are large.

### Comparison of within-collection samples

Based on assumptions about gemmae dispersal ability and clonal growth of independent gametophyte populations, we expected that _C. intricatum_ samples from the same collection would likely be identical clones, but pairwise distances of within-collection samples are not consistently among the lowest. They are generally lower than pairwise distances of samples at larger geographic scales, but span the lower half of the total range of pairwise distances (Figure 12). In no case do sets of samples from the same collection all have their smallest genetic distances to each other. In some cases, the distances that are lower are similar to the distance between the within-collection samples (_e.g_., S09 and S10 have distance 27.1 but both are closer to S15 with distance 25.8 and 26.9, respectively). In other cases other samples have much less genetic distance to one or both of the within-collection samples than the within-collection samples have to each other (_e.g_., S16 and S17 have distance 47.1, but S16 is closer to 17 other samples with distances ranging from 26.9--44.6).

To determine whether this high variation in genetic distances between samples that were expected to be genetically identical is because the samples are not genetically identical or is due to high levels of noise or error relative to the level of genetic signal in the data, we generated a new GBS dataset with simulated clones by replacing the Illumina reads for each set of  within-collection samples with an equal number of reads drawn randomly from a pool of their combined reads. This re-sampled GBS dataset was processed using ipyRAD and loci were filtered using the same settings as the original dataset.  

The distributions of pairwise distances for the not-within-collection samples in the re-sampled and original datasets were similar but the distribution of pairwise distances for the simulated clone samples differed from the original within-collection samples (Figure 15). The distances for simulated clones were lower than most of the original within-collection sample distances, but distances from the original dataset that do fall within the same range as the distances for the simulated clones could potentially represent genetically identical clones. The variation among the pairwise distances for the simulated clones (distance 23.1--29.4) suggests that pairwise distances differing by several distance units cannot be reliably distinguished as different from each other. To explore how this uncertainty affects our ability to infer relationships among samples, we generated an adjusted minimum spanning network showing additional connections between samples with distances that cannot be distinguished from the distance to the least distant sample (Figure 16). The resulting network better reflects the uncertainty in which samples are most similar, with most samples having more than one most similar sample. The network features a cluster of eight Illinois samples with distances suggesting they may be genetically identical or at least very similar to each other. The Western Pennsylvania, Massachusetts, and one Illinois sample (S17) are the most genetically distant from other samples (and are distant from each other as well), while the Eastern Pennsylvania samples and remaining Illinois samples are at intermediate distances to each other and to the closely related Illinois samples.

DISCUSSION
==========

## Separating samples by species using genotyping-by-sequencing data

Analysis of the population genetics of _C. intricatum_ is complicated by the presence of other filmy fern species with partially overlapping ranges and morphologically indistinguishable gametophytes. Limitations in the amount of DNA that can be extracted from tiny amounts of gametophyte tissue and the possibility that multiple extractions from the same gametophyte collection may sample genetically different individuals or even individuals of different species make it challenging to extract enough DNA to use for both species identification and for other analyses. One objective of this study was to determine whether species identification of _C. intricatum_, _V. boschiana_, and _D. petersii_ gametophytes and population genetic analyses of the gametophytes determined to be _C. intricatum_ could both be performed using a single genotyping-by-sequencing dataset.

Structure analyses and neighbor joining trees suggest that in most cases, the species of unknown gametophytes can be inferred from how they group relative to _V. boschiana_ and _D. petersii_ sporophyte samples, and that assuming species based solely on presence or absence of sporophytes at or near a site may lead to misidentification. Seven samples were assigned a different species based on Structure grouping than what was suspected based on geographic location and proximity to sporophytes, including a sample from Georgia found growing with _D. petersii_ sporophytes that appears to be _C. intricatum_ and four samples growing without sporophytes that appear to be _V. boschiana_. Some samples with low read depths could not be unambiguously assigned to species, but four samples with high read depth appear to contain mixtures of more than one species of gametophyte or hybrid gametophytes. Two of these are not completely unexpected--potential _C. intricatum_/_V. boschiana mixtures or hybrids at Southern Illinois sites where _V. boschiana_ sporophytes are in close proximity to many _C. intricatum_ populations. But one potential _C. intricatum_/_D. petersii_ sample was collected in Southern Illinois, far from _D. petersii_ sporophytes, and the other in Arkansas, where all gametophytes were assumed to be _D. petersii_.

Intermingling of species in gametophyte populations and hybridization among filmy fern gametophytes have been observed in Asia [@Ebihara2009-sg] so intermingled species or hybridization of North American filmy ferns is possible, although in previous _C. intricatum_ isozyme analyses, no evidence of individuals with hybrid banding patterns have been detected [@Farrar1985-nm] and the cpDNA based markers that have recently been used to distinguish between North America filmy fern gametophytes [@Pinson2017-eu] would not be sufficient to detect hybrids. Further work is needed to determine if these samples represent mixtures of species, gametophytes from hybrids or just did not have sufficient genetic signal to consistently assign them to species.

## Population genetics of _C. intricatum_

A previous attempt to explore the population genetics of _C. intricatum_ failed to find variability with isozyme markers [@Farrar1985-nm], and we failed to find sufficiently variable cpDNA markers, but genotyping-by-sequencing allowed us to identify relatively large numbers of variable loci to evaluate patterns of variation. There is genetic variability at all geographic scales. Individuals growing in contiguous patches are sometimes--but not always--genetically similar, and pairwise genetic distances also vary drastically within sites, within regions, and between regions. There is only weak evidence of isolation by distance and no clear geographic structuring of genetic variation.

Noise within the data makes it difficult to distinguish clones from genetically similar individuals--no two samples have identical multi-locus genotypes--but eight samples (collected from sites throughout Southern Illinois) have pairwise distances similar to those of simulated clones, suggesting this may be a regionally widespread clone. It is possible that there are widespread clones in other regions (or that there are other widespread clones in Southern Illinois) but they were not included in our sampling. We cannot rule out long-distance gemmae dispersal but there is no evidence of clones present in more than one region, which is consistent with assumptions about limited gemmae dispersal distances. Multiple genetically distant genotypes within five of the six sites with multiple samples suggest movement between sites or that sites were colonized by more than a single propagule.

Sampling depth in the glaciated areas and nearby areas is not sufficient to make statistical comparisons of diversity or determine whether they were colonized following a stepping-stone or leading-edge pattern. The sample most geographically distant from other samples (S36 in Massachusetts) is also one of the most genetically distant from other samples, but the samples in glaciated areas in Pennsylvania are not most similar to samples from the nearest non-glaciated sites included in this study. Samples within a glaciated site with more than one sample are not closely related to each other, suggesting colonization by propagules from multiple sites, colonization by multiple propagules from a diverse site, or high levels of gene flow after colonization.

## Evaluation of hypotheses for establishment and maintenance of _C. intricatum_ independent gametophyte populations

Hypotheses for how independent gametophyte populations of _C. intricatum_ were established and reached their current distribution in Eastern North America can be categorized first, by whether sporophytes and local spore dispersal were ever present in North America (Table 4). Those hypotheses that include North American sporophytes can be further divided by when the sporophytes were in North America (anciently or recently and perhaps ongoing), and the hypotheses that do not include North American sporophytes can be divided based on when the propagules establishing the North American gametophytes populations arrived (again, anciently or recently and perhaps ongoing).

What can we predict about the patterns of genetic diversity under each of these hypotheses?

Pleistocene relicts: Some unknown pattern of diversity existed anciently. After sporophytes died out and spores stopped being produced, clonal competition and genetic drift would decrease within-site diversity and increase between-site diversity, whereas gemmae dispersal would have opposite effect. Mutation would be the only source of new genotypes, they would be similar to existing ones, and most likely be found close to where they originated. Glaciated sites would likely have reduced diversity due to founder effects and be most similar to populations near the glacial boundary.

Cryptic sporophytes: Rare sexual reproduction and spore dispersal would counteract loss of genotypes due to clonal competition and genetic drift, and increase within-site diversity. Long distance dispersal would be more likely, increasing gene flow and reducing between-site diversity. Many new genotypes could be generated sexually, they could be quite different from the parental genotypes, and they could be distributed over greater distances. Diversity in glaciated sites could vary based on frequency of spore dispersal, but might still have reduced diversity if they are at the limits of the range.

Ancient spore invasion: Patterns of diversity would depend on the number of introductions, the genetic variation present in the source, levels of gemmae dispersal post-invasion, and the mutation rate. A single introduction with subsequent asexual reproduction could lead to low levels of within and between-site diversity whereas multiple introductions from a diverse source could lead to complex patterns of variation. Diversity in glaciated sites would depend on these same factors but likely similar to that expected under the Pleistocene relicts hypothesis, since these sites would be colonized by gemmae after the ancient spore invasion.

Separation of generations: Patterns of variation would depend on the same factors as in an ancient spore invasion but with presumably fewer new genotypes generated by mutation and possibly less difference between glaciated and non-glaciated sites, since they have as much opportunity to be established by invading spores as any other site.

The predictions based on each of the hypotheses can vary drastically based on unknown factors and it is not clear that the pattern of variation identified for _C. intricatum_ is entirely inconsistent with any of them. It does not appear to be a single spore introduction (or very few introductions) with subsequent dispersal of only gemmae--there are multiple genetically distant genotypes in each region--but we cannot rule out scenarios with many introductions from a diverse source.

## Caveats and challenges

There are challenges encountered in this study that may impact or limit inferences.

The small size and intertwining growth of individual gametophytes required extracting DNA from samples containing mixtures of individuals, which were assumed to be clones but in some cases appear to contain multiple genotypes. There are methods available for analyzing data from pools of individuals but they require similar amounts of DNA from each individual in a pool and similar numbers of individuals in each pool [@Anand2016-zn], which is not the case in this these samples. Combining individuals with two or more unique genotypes would lead to overestimates of heterozygosity and inference of a mixed genotype that is genetically distant from both underlying genotypes, or even from another sample with the same genotypes mixed in different ratios. Low read depths exacerbate this effect. If individuals growing intertwined with each other are clones or genetically similar then mixing individuals should not affect inferences, but this may not always be the case.

Because we assumed that most, if not all, of the samples from single collections would be clones we did not include direct replicates in our GBS library, making it difficult to determine the level of noise in the dataset. Using simulated clones in a re-sampled GBS library allowed us to estimate noise and determine that some samples are very similar to each other (possibly clones) whereas other samples are more genetically distant.

The difficulty of getting sufficient DNA from tiny tissue samples also biased our sampling toward larger collections from larger populations, which might be expected to contain more diversity. Sampling was reduced further by samples determined to be other species or samples with low readcounts that were excluded from the _C. intricatum_ analysis. Additional sampling would allow better analysis of patterns of variation throughout the range, and more dense sampling in the northeastern part of the range would allow meaningful comparisons of diversity in previously glaciated and non-glaciated areas.

Using multiple gametophytes in each sample and focusing on larger collections complicates interpretation of this dataset, but was necessary given the limited DNA that can be extracted from a single contiguous chain of gametophyte cells and the fragmenting of individuals that occurs in preserved samples. These same challenges likely affected isozyme and cpDNA studies of _C. intricatum_ as well, but the low or non-existent variation found using those markers conceals the problem. A method for generating highly variable markers from the tiny amounts of DNA that can be extracted from a single gametophyte, or tissue banks grown from single gametophytes or gemmae (a slow process) are needed for population genetics studies of independent fern gametophytes.

##  Conclusions and further work

Many of the expectations for genetic diversity under different hypotheses are based on untested assumptions about the dispersal ability of _C. intricatum_ gemmae relative to spores based on their size. Dispersal of asexual or vegetative propagules has been better studied in bryophytes. Bryophytes that rarely or never produce sporophytes are common--sporophytes are unknown for 20% of British mosses--and bryophyte propagules with size and structure comparable to _C. intricatum_ gemmae play an important role in habitat colonization and local or regional maintenance of populations [@Frey2011-uy]. Dispersal mechanisms including water, wind, and animal vectors have been demonstrated in bryophytes [@Pasiche-Lisboa2018-kz] but nothing is known about the dispersal mechanisms or ability of Crepidomanes gemmae or gametophyte fragments. The lack of gametophyte populations in recently disturbed sites like train tunnels or roadcuts is taken as evidence of poor dispersal [@Farrar1992-pm] but it is not clear whether these habitats have been extensively surveyed or whether there are habitat requirements differences that make those sites unsuitable. Experiments like the recent study to determine limiting factors in _V. appalachiana_ distribution [@Chambers2016-kf] could test this.

Nothing in this study addressed the question of why we do not find _C. intricatum_ sporophytes, and it is unlikely that population genetics data can provide an answer--though the presence of multiple genotypes in some populations suggests that it is not a problem of self-incompatibility in clonal populations. Further work is needed to investigate potential explainations for the lack of sexual reproduction in _C. intricatum_.

We were not able to use GBS data to differentiate among hypotheses for how _C. intricatum_ independent gametophyte populations were established and reached their current distribution, but this study provides useful information on filmy fern independent gametophytes in Eastern North America. The data presented highlight the risks of assuming gametophyte species based on geographic location and presence or absence of sporophytes--all three species included in this study may be found without sporophytes and _C. intricatum_ may be found growing with sporophytes of other filmy fern species. A contiguous patch of independent gametophytes may contain more than a single species, which is significant based on suggestions that some Asian independent gametophytes are not directly producing sporophytes, but may be contributing gametes for hybridization [@Ebihara2009-sg].

This study provides a first look at patterns of genetic variation in _C. intricatum_ gametophyte populations, which demonstrate a lack of genetic structure, only weak evidence of isolation by distance, and genetic variation present at all geographic scales. Even within a contiguous patch of gametophytes, individuals growing intimately entwined may not be genetically identical clones.


ACKNOWLEDGEMENTS
================
NOTE- U of U CHPC for computing resources, Zach Gompert for guidance on library prep


REFERENCES
==========
