---
title: Bad working title--Population genetics of independent gametophytes using genotyping-by-sequencing
author: Aaron M. Duffy, Don R. Farrar, Paul G. Wolf
---

NOTES
=====

* What about Paul's suggestion that some of the noise in the data could be due to loci from contaminants? BLAST locus consensus sequences? I did this with my npgeno loci. With no fern genomes BLAST is going to find hits in other genomes. How do we decide how much similarity to potential contaminants is too much?
* How does distance between samples compare to distance within a sample? For my replicates, what is the distance between each pair? How much of that distance is due to differences that are autapomorphic? How does that compare to the distance between non-replicate pairs? Maybe show some of this by plotting pairwise distances (by what?) and labeling the replicate comparisons.
* Perhaps the IBD will show a general trend now that the non-Crep signal is gone?
* Now that we have a better idea of Crep vs Mixed vs Non-Crep species assignments for the samples (including considerations of samples with very few loci), go back and see how accurate the PCR-based species ID method worked. Is that something we could propose as a quick species-test for independent gametophytes?

ABSTRACT
========

INTRODUCTION
============

Things to include (needed for methods/results to make sense):

This is an example citation [@Altschul1990-ri; @Andrews2010-ov].

METHODS
=======

Secondary heading
-----------------

Hashtag-style headings work too if you need further levels. Some text.

RESULTS
=======

DISCUSSION
==========

REFERENCES
==========
